src/app/shared/header/header.component.html

```diff
@@ -19,7 +19,10 @@
 <header class="mdm-header">
   <div class="mdm-header__logo">
     <a [routerLink]="logoLink?.routerLink ?? ''">
-      <img [src]="logoLink?.imageSrc" [alt]="logoLink?.label" />
+      <img
+        [src]="logoLink?.customImageSrc || logoLink?.defaultImageSrc"
```

Comment:
By introducing a possible custom logo, we should ensure that the original "Mauro Data Mapper" logo is always guaranteed to appear somewhere on the page - in the `mdm-ui` is always appears in the footer for instance, so I think that `mdm-explorer` footer should be updated too. I'm not sure if that is in scope for this task or should be spun out to another task.

Reply:
I would prefer this to be addressed in a follow-up task.

Reply:
Created #310